public/data/devesh2511.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Devesh Jain",
+  "type": "personal",
+  "bio": "Web Developer | Open Source Enthusiast | Computer Science Sophomore",
+  "avatar": "https://github.com/devesh2511.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/devesh2511",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/devesh2511",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

I think it might look better if you capitalize the `i` 😅 What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```